docs/tutorials/ble/scanning.mdx

```diff
@@ -24,17 +26,18 @@ The following example shows how to scan for 3 seconds for the addresses of remot
 ```toit
 import ble
 
-BATTERY_SERVICE ::= ble.uuid 0x180F
+BATTERY_SERVICE ::= ble.BleUuid "180F"
```

Comment:
Again, above this: e.g. implement a specific service should be e.g. advertising a specific service. 
And in the text, if you create your own full 128-bit UUID you can add your custom service.

Reply:
done.